onyo/lib/tests/test_commands_tree.py

```diff
@@ -59,15 +59,36 @@ def test_onyo_tree_single(inventory: Inventory,
     assert inventory.repo.git.is_clean_worktree()
 
 
+@pytest.mark.ui({'yes': True})
+def test_onyo_tree_dirs_only(inventory: Inventory,
+                             capsys) -> None:
+    r"""Display directory structure w/o any assets or other files"""
```

Comment:
Nit: this comment conflates assets with files (ignoring asset dirs, which are allowed).